Mathlib/Data/Fin/Basic.lean

```diff
@@ -1296,7 +1296,7 @@ theorem le_sub_one_iff {n : ℕ} {k : Fin (n + 1)} : k ≤ k - 1 ↔ k = 0 := by
   simp
 #align fin.le_sub_one_iff Fin.le_sub_one_iff
 
-@[simp]
+@[simp, nolint simpNF] -- lint timeout
```

Comment:
This is a bit concerning. Any idea why it's occurring?

Reply:
It's not needed now : )